### ISSUE TITLE
Update for "Ghostcat" Tomcat Vulnerability?

### ISSUE THREAD

**Issue by MathewBiddle** [axiom-data-science/docker-erddap#9]
It looks like the docker image for latest and v2.02 at https://hub.docker.com/r/axiom/docker-erddap/tags have been updated to address the Tomcat vulnerability. Is there a plan to update this repository as well?

**Comment by kwilcox** [comment]
Hey Matt!  This repository produces the images at https://hub.docker.com/r/axiom/docker-erddap. I had to manually regenerate them because the upstream images (`unidata/tomcat-docker:8.5`) doesn't version their image. While I don't 100% agree with not adding some sort of indication that the image was updated to the tag (`unidata/tomcat-docker:8.5-[increment]`) it is common practice to do what they do and just regen an image with the same tag.

There is nothing to update in this repository, if someone were to build this manually they should `docker pull unidata/tomcat-docker:8.5` first.
